cmd/edge-orchestration/capi/main.go

```diff
@@ -139,6 +139,7 @@ var (
 	orcheEngine orchestrationapi.Orche
 )
 
+// OrchestrationInit runs orchestration service and discovers remote orchestration services
 //export OrchestrationInit
```

Comment:
How about removing this line?

Reply:
We got a [warnings](https://goreportcard.com/report/github.com/lf-edge/edge-home-orchestration-go#golint) from the Go Report Card

Reply:
Do we need `//export OrchestrationInit` line?

Reply:
Without `//export OrchestrationInit` line, we can't create `orchestration.h` file correctly.